src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,34 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.save(actor);
+            transaction.commit();
+        } catch (RuntimeException e) {
+            transaction.rollback();
+            throw new DataProcessingException("Error when adding a movie", e);
+        } finally {
+            session.close();
+        }
+        return actor;
     }
 
     @Override
     public Optional<Actor> get(Long id) {
-        return null;
+        Session session = null;
+        Actor actor = null;
+        try {
+            session = factory.openSession();
+            actor = session.get(Actor.class, id);
+        } catch (RuntimeException e) {
+            throw new DataProcessingException("Error when adding an actor", e);
```

Comment:
do you think it's a good message for such method?
the same for other similar methods